pkg/operator/sync.go

```diff
@@ -151,6 +151,12 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 		return err
 	}
 
+	additionalTrustBundle, err := optr.getCAsFromConfigMap("openshift-config-managed", "user-ca-bundle", "ca-bundle.crt")
```

Comment:
again this config map is only present when requested by the user.

Reply:
fixed